packages/flutter/lib/src/painting/text_style.dart

```diff
@@ -475,6 +475,12 @@ class TextStyle with Diagnosticable {
   /// The `package` argument must be non-null if the font family is defined in a
   /// package. It is combined with the `fontFamily` argument to set the
   /// [fontFamily] property.
+  ///
+  /// On Apple devices the strings 'CupertinoSystemText' and
+  /// 'CupertinoSystemDisplay' are used as proxies for the Apple system fonts.
```

Comment:
mention `[fontFamily]` in this sentence somehow to link to it and give people context where CupertinoSystemText and CupertinoSystemDisplay can be used.

Reply:
What happens if you use these magic strings on non-Apple devices? Maybe document that as well